commit 195d74be717af14e5991f818f73f067367bfc1ed
Author: Zhao Gongyi <zhaogongyi@huawei.com>
Date:   Mon Sep 5 21:36:13 2022 +0800

    selftests/cpu-hotplug: Delete fault injection related code
    
    Delete fault injection related code since the module has been deleted.
    
    Signed-off-by: Zhao Gongyi <zhaogongyi@huawei.com>
    Signed-off-by: Shuah Khan <skhan@linuxfoundation.org>

diff --git a/tools/testing/selftests/cpu-hotplug/config b/tools/testing/selftests/cpu-hotplug/config
deleted file mode 100644
index d4aca2ad5069..000000000000
--- a/tools/testing/selftests/cpu-hotplug/config
+++ /dev/null
@@ -1 +0,0 @@
-CONFIG_NOTIFIER_ERROR_INJECTION=y